jina/clients/request/helper.py

```diff
@@ -67,4 +67,5 @@ def _add_docs_groundtruths(req, batch, data_type, _kwargs):
 
 
 def _add_ids(req, batch):
-    req.ids.extend(batch)
+    string_ids = (str(doc_id) for doc_id in batch)
```

Comment:
why is this needed? id is supposed to be already a str?

Reply:
I think it can be an integer. Or where is it converted before?
Let me check if this is needed.

Reply:
Okay, I checked it.
If I run
```index_flow.delete(input_fn=range(0,3))```
then I get the ids as integers. So they are not converted.
![image](https://user-images.githubusercontent.com/11627845/106958849-3337ef80-673a-11eb-9366-cdfe8ad0eff9.png)

Reply:
ok so are we allowing that right now?

Reply:
Yes, we allow to delete by id.
In addition, since we allow integer ids for the constructor when creating a document, we allow integers for deleting documents as well. Otherwise this might become a source of client site errors.
The int to str conversion is now very simplistic it ist just `str(num)`